src/Microsoft.FeatureManagement/AssemblyInfo.cs

```diff
@@ -1,4 +1,4 @@
-﻿﻿// Copyright (c) Microsoft Corporation.
+﻿// Copyright (c) Microsoft Corporation.
```

Comment:
I am curious about what is the difference of this change? Which rule triggered it?